x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -212,20 +205,16 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
             .filter((result) => result[group].isNoData)
             .map((result) => buildNoDataAlertReason({ ...result[group], group }))
             .join('\n');
-        } else if (nextState === AlertStates.ERROR) {
```

Comment:
@simianhacker Don't we have an error state anymore?

Reply:
Not in the same sense. There was this error condition that existed in the old implementation that "no longer" exists in the new methodology. Errors from this point forward are going to be exceptions caught by the framework.